### PR TITLE
feat(simple_object_merger): add simple_object_merger

### PR DESCRIPTION
**Pull request by scepter914**
## Description

This PR add `simple_object_merger`, which merge DetectedObjects without matching process.
This package aim to use for multiple radar detection.
About detail algorithm and purpose I wrote to README.md, so it would be helpful if you could review it at same time.

## Related links

Nothing

## Tests performed

Tested by rosbag.
I tested using topics of `/perception/object_recognition/detection/radar/clustered_objects` and `/perception/object_recognition/detection/lidar/objects`.

## Notes for reviewers

Nothing

## Interface changes

Nothing

## Effects on system behavior

Nothing

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/simple_object_merger/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+This package can merge multiple DetectedObjects without matching processing.
+[Object_merger](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/object_merger) solve data association algorithm like Hungarian algorithm for matching problem, but it needs computational cost.
+In addition, for now, [object_merger](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/object_merger) can handle only 2 DetectedObjects topics and cannot handle more than 2 topics in one node.
+To merge 6 DetectedObjects topics, 6 [object_merger](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/object_merger) nodes need to stand.
+
+So this package aim to merge DetectedObjects simply.
+This package do not use data association algorithm to reduce the computational cost, and it can handle more than 2 topics in one node to prevent launching a large number of nodes.
+
+### Limitation
+
```
Comment (YoshiRi) [inline perception/simple_object_merger/README.md:18]: I think this package has some assumptions:
- all input object has same(close) timestamp
- none of inputs will drop

It's better to write down these assumptions in README.

**Review by YoshiRi — COMMENTED** [review]
## risks in this algorithm

When one of the sensor objects drops or delayed, this packages will concatenate wrong pairs of the input objects. 
I think we need to take care of this issue.
@miursh  how do you think about it?

## basic idea

- objects with large difference in timestamps should not be merged
- old, not completely gathered objects should be discarded or published

## possible solutions

I think watch timestamps of objects and do sanity check based on timestamps in onTimer() will ok, but let me think while...

File: perception/simple_object_merger/README.md (hunk 1 above)
Comment (scepter914) [reply perception/simple_object_merger/README.md:18]: > To care sensor objects drops or delayed

I think setting timeout parameter is reasonable for this package and error state should be handled by diagnostic layer.

> all input object has same(close) timestamp

Some radar run independently, so this assumption cannot be satisfied.

Comment (scepter914) [reply perception/simple_object_merger/README.md:18]: I added timeout parameter and update README.md at [a9fb023](https://github.com/autowarefoundation/autoware.universe/pull/4219/commits/a9fb02376cfb04cd9e32abe70808a819acc04e57)

**Review by YoshiRi — APPROVED** [review]
LGTM

**Pull request merged**